packages/office-ui-fabric-react/src/components/TextField/TextField.Props.ts

```diff
@@ -24,6 +24,12 @@ export interface ITextField {
    * @param end Index of the end of the selection.
    */
   setSelectionRange: (start: number, end: number) => void;
+
+  /** Gets the selection start of the text field */
+  selectionStart: number;
+
+  /** Gets the selection end of the text field */
+  selectionEnd: number;
```

Comment:
Add "returns -1 if no selection found" to both of these descriptions. The `value` prop returns undefined when no value present, so this deviates a little bit.